executor/executor.cc

```diff
@@ -1308,6 +1309,15 @@ uint32* write_output(uint32 v)
 	return output_pos++;
 }
 
+uint32* write_output_64(uint64 v)
+{
+	if (output_pos < output_data || (char*)output_pos >= (char*)output_data + kMaxOutput)
+		fail("output overflow: pos=%p region=[%p:%p]",
+		     output_pos, output_data, (char*)output_data + kMaxOutput);
+	*output_pos = v;
```

Comment:
in pkg/ipc/ipc.go we do:
```
func readUint64(outp *[]byte) (uint64, bool) {
	out := *outp
	if len(out) < 8 {
		return 0, false
	}
	v := binary.LittleEndian.Uint64(out)
	*outp = out[8:]
	return v, true
}
```
that's native endianess, right.
Or, do you plan more changes to pkg/ipc as well?

Reply:
added more ipc stuff